client/src/app.tsx

```diff
@@ -226,10 +226,10 @@ export function App(appProps) {
             )}
             {PLUS_IS_ENABLED && (
               <Route
-                path="/offline-settings"
+                path="/settings"
                 element={
                   <StandardLayout>
-                    <OfflineSettings {...appProps} />
+                    <Settings {...appProps} />
                   </StandardLayout>
```

Comment:
Should this be here at all? It means we have settings at https://developer.allizom.org/en-US/settings and https://developer.allizom.org/en-US/plus/settings